### PR TITLE
Save camera angle and improve number formatting

### PR DESCRIPTION
**Pull request by hungnguyen94**
This pull request allows you to save the camera angle. It also improves the number formatting, because they are all doubles now. Doubles are limited to 2 numbers after comma.

### CODE REVIEW

File: src/main/webapp/app/entities/camera/camera-dialog.html [hunk 1]
```diff
@@ -60,6 +60,26 @@ <h4 class="modal-title" id="myCameraLabel">Create or edit a Camera</h4>
...
         </div>
         <div class="form-group">
+            <label class="control-label" for="field_angle">Angle</label>
+            <input type="number" class="form-control" name="angle" id="field_angle"
+                   ng-model="vm.camera.angle"
+                   min="0" max="360" />
+            <div ng-show="editForm.angle.$invalid">
+                <p class="help-block"
+                   ng-show="editForm.angle.$error.min" translate-value-min="0">
+                    This field should be more than 0.
```
Comment (rubenwiersma) [inline src/main/webapp/app/entities/camera/camera-dialog.html]: You could change this to: The angle should be more than 0.

File: src/main/webapp/app/entities/camera/camera-dialog.html [hunk 2]
```diff
@@ -60,6 +60,26 @@ <h4 class="modal-title" id="myCameraLabel">Create or edit a Camera</h4>
...
+                </p>
+                <p class="help-block"
+                   ng-show="editForm.angle.$error.max">
+                    This field cannot be more than 100.
```
Comment (rubenwiersma) [inline src/main/webapp/app/entities/camera/camera-dialog.html]: Shouldn't this be: The angle cannot be bigger than 360?

**Comment by rubenwiersma** [comment]
Small change, so not a lot to comment. But I did find something 😉

**Comment by hungnguyen94** [comment]
Fixed the error messages :wink:

**Comment by ChrisBerg** [comment]
Nice, looks good!

**Pull request merged**